content/en/workflows/actions_catalog/generic_actions.md

```diff
@@ -4,7 +4,88 @@ kind: documentation
 disable_toc: false
 type: workflows
 further_reading:
-- link: "logs/processing/pipelines"
+- link: "/integrations/"
   tag: "Documentation"
-  text: "Log processing pipelines"
+  text: "Learn about integrations"
 ---
+
```

Comment:
Use the beta frontmatter and shortcode described in docs4docs.

Reply:
Thanks, I updated the frontmatter, but posted a question about the shortcode previously